nasa/harmony
# DAS-1604 - Add UAT service chain for HyBIG.

## Jira Issue ID

DAS-1604

## Description

This PR adds a service chain for the Harmony Browse Image Generator (HyBIG).

_Side note:_

I spent a while trying to get the tests to pass, because the new-ish `/docs` endpoint wasn't rendering correctly. It turns out that I now needed to explicitly specify whether the service chain has bounding box subsetting capabilities or not (I had initially just not said anything assuming that was the same as saying `bbox: false`). I'm not sure if that's documented anywhere, but it seems like a subtle change that might be worth calling out.

## Local Test Steps

* Pull this branch.
* Use the `bin/build-image` script in the HyBIG repo to build the service image.
* Update your `.env` file to include `hybig` as one of the `LOCALLY_DEPLOYED_SERVICES`.
* Rebuild your local Harmony instance.
* Run the request below. It should route to `sds/HyBIG` (as seen in the `/workflow-ui`). Note the request output isn't important, as the service itself is not fully implemented.

```
http://localhost:3000/C1256584478-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&maxResults=1&format=image%2Fpng
```

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
